std/net/curl.d

```diff
@@ -627,6 +627,60 @@ unittest
     assert(res == cast(ubyte[])[17, 27, 35, 41]);
 }
 
+/* Encode associative array using www-form-urlencoding
+ * perhaps this should live in std.uri
```

Comment:
> Please note that I am not an experienced d developer. 

I write my first line of D three months ago - you will get there!
Usually it helps to have a look at other method headers. Feel free to ask #d - there are usually enough people around to help you. You can ping me (greenify) too ;-)

Have a look here: https://wiki.dlang.org/Access_specifiers_and_visibility

> Package extends private so that package members can be accessed from code in other modules that are in the same package. This applies to the innermost package only, if a module is in nested packages.

It's just `package char[] post() ...`